test/test_runner.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Get current test file name
+TESTFILE_NAME=${PGAPPNAME##pg_regress/}
+# Set different name for each test database
+# As pg_regress does not support cleaning db after each test
+TEST_CASE_DB="ldb_${TESTFILE_NAME}"
+
+# Drop db after each test on exit signal
+function drop_db {
+  cat <<EOF | psql "$@" -U postgres -d postgres -v ECHO=none >/dev/null 2>&1
+    SET client_min_messages=ERROR;
+    DROP DATABASE "${TEST_CASE_DB}";
+EOF
+}
+
+trap drop_db EXIT
+
+# install lanterndb extension
+psql "$@" -U postgres -d postgres -v ECHO=none -q -c "DROP DATABASE IF EXISTS ${TEST_CASE_DB};" 2>/dev/null
+psql "$@" -U postgres -d postgres -v ECHO=none -q -c "DROP DATABASE IF EXISTS ${TEST_CASE_DB};" 2>/dev/null
+psql "$@" -U postgres -d postgres -v ECHO=none -q -c "CREATE DATABASE ${TEST_CASE_DB};" 2>/dev/null
```

Comment:
From [postgres docs](https://www.postgresql.org/docs/current/regress-run.html)
> When using installcheck mode, these tests will create and destroy test databases whose names include regression, for example pl_regression or contrib_regression. Beware of using installcheck mode with an installation that has any non-test databases named that way.

It seems the upstream approach is to create and restroy databases hosting the tests, in stead of using an existing database. We should follow the same convention.

Reply:
Yes actually we are creating and destroying the database after each test case, I just put `IF EXISTS` here, so if the script will be terminated in the middle it won't cause any issues in next run.